build-system/tasks/runtime-test/runtime-test-base.js

```diff
@@ -116,82 +117,21 @@ class RuntimeTestConfig {
   }
 
   /**
-   * Updates the browsers based off of the test type
-   * being run (unit, integration, a4a) and test settings.
-   * Defaults to Chrome if no matching settings are found.
+   * Picks a browser config based on the the test type and command line flags.
+   * Defaults to Chrome.
    */
   updateBrowsers() {
     if (argv.edge) {
-      Object.assign(this, {
-        browsers: [argv.headless ? 'EdgeHeadless' : 'Edge'],
-      });
-      return;
+      Object.assign(this, {browsers: ['EdgeCustom'], customLaunchers});
+    } else if (argv.firefox) {
+      Object.assign(this, {browsers: ['FirefoxCustom'], customLaunchers});
+    } else if (argv.ie) {
+      Object.assign(this, {browsers: ['IECustom'], customLaunchers});
+    } else if (argv.safari) {
+      Object.assign(this, {browsers: ['SafariCustom'], customLaunchers});
+    } else {
+      Object.assign(this, {browsers: ['ChromeCustom'], customLaunchers});
```

Comment:
Will it be easier to read to use a switch statement, or if statements with early returns?

Reply:
Good question. Early returns made the function unnecessarily verbose, and switch statement didn't work because the key in `argv` is different for each case. But we can eliminate all the `Object.assign` repetition (done in last commit). 👍